src/htmlparser.js

```diff
@@ -431,138 +427,3 @@ class HTMLParser {
     }
   }
 }
-
-exports.HTMLParser = HTMLParser;
-exports.HTMLtoXML = function (html) {
```

Comment:
Are these functions unused and undocumtented? Because I'm unsure why we remove them here. Or maybe I oversaw something.

Reply:
> Are these functions unused and undocumtented?

Yeah, both. They are used no-where and mentioned no-where.

Reply:
May be let's not remove them yet, till something is conclusive?

Reply:
Removed for now. lets deal with that later.